ddl/constant.go

```diff
@@ -55,7 +55,7 @@ const (
 		ddl_job_id bigint not null,
 		ele_id bigint not null,
 		ele_key blob,
-		store_id bigint,
+		ddl_physical_tid bigint,
```

Comment:
Is there any risk here? Changed the meta of the table. And maybe we need change it back later.

Reply:
No, it won't be used later, but will be put into meta if needed